modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/export_form_instance.jspf

```diff
@@ -85,6 +98,19 @@ boolean showCSVWarning = StringUtil.equals("enabled-with-warning", csvExport);
 				else {
 					csvWarning.hide();
 				}
+
+				showXLSWarning();
+			};
+
+			var showXLSWarning = function () {
+				var xlsWarning = A.one('#<portlet:namespace />xlsWarning');
+
+				if (fileExtensionSelect.val() === 'xls' && <%= showXLSWarning %>) {
```

Comment:
@samanzanedo, do we really need the variable `showXLSWarning`?

From what I could see, we just need to check if `fileExtensionSelect.val() === 'xls'`, right?

In addition, I think we should create a new method to handle the display of the warning messages instead of make method `showCSVWarning` invoke the method `showXLSWarning`. I see 2 ways to make it happen:

1. Rename the method `showCSVWarning` to `showWarnings` and add the logic of the method `showXLSWarning` to the renamed method.
2. Create a new method named `showWarnings` which will invoke the methods `showCSVWarning` and `showXLSWarning`.

IMHO, I see the second alternative better for code legibility.